### PR TITLE
update docs and install script for helm-dashboard

### PR DESCRIPTION
**Pull request by JacieChao**
- add helm-dashboard for Dockerfile(https://github.com/cnrancher/autok3s/issues/499)
- add helm-dashboard for install script(https://github.com/cnrancher/autok3s/issues/499)
- update docs for helm-dashboard(https://github.com/cnrancher/autok3s/issues/499)
- add troubleshooting with k3d memory setting(https://github.com/cnrancher/autok3s/issues/604)

### CODE REVIEW

File: docs/i18n/en_us/alibaba/README.md [hunk 1]
```diff
@@ -356,16 +356,34 @@ AutoK3s support [cnrancher/kube-explorer](https://github.com/cnrancher/kube-expl
...
+```bash
+autok3s helm-dashboard --port 8888
+```
+
+After the server started success, you can access the helm-dashboard by `http://127.0.0.1:8888`
+
+You can also enable/disable helm-dashboard any time from UI, and access helm-dashboard by `dashboard` button.
+
+![](../../../assets/enable-helm-dashboard.png)
+![](../../../assets/access-to-helm-dashboard.png)
```
Comment (Jason-ZW) [inline docs/i18n/en_us/alibaba/README.md:387]: 这里是不是换成helm-dashboard的logo能好一些？或者官方叫法。dashboard感觉又些混淆？

Comment (JacieChao) [reply docs/i18n/en_us/alibaba/README.md:387]: fixed

**Review by Jason-ZW — APPROVED** [review]
LGTM

**Review by orangedeng — APPROVED** [review]
LGTM

**Pull request merged**